package.json

```diff
@@ -22,6 +22,10 @@
     "build": "npm-run-all -p build:*",
     "build:css": "node-sass src/plugin.scss dist/videojs-errors.css --output-style=compressed --linefeed=lf",
     "build:js": "rollup -c scripts/rollup.config.js",
+    "prebuild:lang": "npm-run-all -s prebuild:lang:*",
```

Comment:
this should be updated to whatever we agree in https://github.com/videojs/video.js/pull/6149

Reply:
Done.